app/views/layouts/_topbar.html.haml

```diff
@@ -14,7 +14,7 @@
     %ul.right
       %li.active
         - if user_signed_in?
-          %a{href: "#", id: "#{current_user.id}", class: "username"}= "Signed in as #{current_user.username}"
+          %a{ href: "users/#{current_user.id}", id: "#{current_user.id}", class: "username" }= "Signed in as #{current_user.username}"
```

Comment:
Line is too long. [134/80]<br>Avoid defining `class` in attributes hash for static class names<br>`= "..."` should be rewritten as `...`